torch/testing/_internal/common_methods_invocations.py

```diff
@@ -142,7 +147,10 @@ def get_inplace(self):
 
     def sample_inputs(self, device, dtype, requires_grad=False):
         """Returns an iterable of SampleInputs."""
-        return tuple()
+        if self.sample_inputs:
```

Comment:
Probably, here it should be
```diff
- if self.sample_inputs
+ if self.sample_inputs_func
```

Reply:
I updated these in https://github.com/pytorch/pytorch/pull/48828.